src/main/java/org/jmhsrobotics/frc2024/subsystems/LED/LEDSubsystem.java

```diff
@@ -0,0 +1,67 @@
+package org.jmhsrobotics.frc2024.subsystems.LED;
+
+import edu.wpi.first.wpilibj.AddressableLED;
+import edu.wpi.first.wpilibj.AddressableLEDBuffer;
+import edu.wpi.first.wpilibj.RobotBase;
+import edu.wpi.first.wpilibj.simulation.AddressableLEDSim;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class LEDSubsystem extends SubsystemBase {
+	private AddressableLED LED;
```

Comment:
rename to lowercase led. Names with capital letters are generally used for Classes and Constants.